src/assets/icon/PostToCommunity.js

```diff
@@ -23,7 +24,7 @@ const PostToCommunity = (props) => (
       clipRule="evenodd"
       d="M63.822 6.13a3.861 3.861 0 0 1 5.46 0l4.587 4.588a3.861 3.861 0 0 1 0 5.46L54.844 35.205a3.861 3.861 0 0 1-1.964 1.054l-5.752 1.165a3.861 3.861 0 0 1-4.551-4.55l1.165-5.753a3.861 3.861 0 0 1 1.054-1.964L63.822 6.131ZM47.192 29.532l.208-1.023 4.09 4.092-1.022.207-4.108.832.832-4.108Zm7.277.587-4.587-4.588L63.82 11.591l2.73-2.73 1.858 1.857.873.873 1.857 1.857-2.73 2.73-13.939 13.94Z"
       fill="#fff"
-      stroke={COLORS.signed_primary}
+      stroke={props.color ?? COLORS.signed_primary}
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change here allows the stroke color of the SVG icon to be customizable via a prop, with a fallback to `COLORS.signed_primary`. This is a good practice as it makes the component more reusable and adaptable to different contexts.
```diff
-      stroke={COLORS.signed_primary}
+      stroke={props.color ?? COLORS.signed_primary}
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->